src/components/Dashboard.jsx

```diff
@@ -1,41 +1,28 @@
-import { useEffect, useState } from 'react';
+import { useState } from 'react';
+import { useAuth } from './AuthProvider';
 import VerticalTabs from './VerticalTabs';
-import PIESCheckin from './tabs/PIESCheckin';
-import Growth from './tabs/Growth';
-import Tips from './tabs/Tips';
-import Events from './tabs/Events';
-import Memberships from './tabs/Memberships';
-import Favorites from './tabs/Favorites';
+import { tabMap, defaultTabs, roleExtraTabs } from './tabsConfig';
 
 export default function Dashboard() {
-    const [active, setActive] = useState(() => {
-        return localStorage.getItem('activeTab') || 'PIES Checkin';
-    });
+    const { role } = useAuth();
 
-    useEffect(() => {
-        localStorage.setItem('activeTab', active);
-    }, [active]);
+    const allKeys = [
+        ...defaultTabs,
+        ...(roleExtraTabs[role] || []),
+    ];
 
-    const tabs = {
-        'PIES Checkin': <PIESCheckin />,
-        'Tips': <Tips />,
-        '⭐ Favorites': <Favorites />,
-        'Growth': <Growth />,
-        'Events': <Events />,
-        'Memberships': <Memberships />,
-    };
-
-    const role = 'owner'; // Placeholder for future role logic
```

Comment:
This was the first clear signal that the “Production” tab wasn’t rendering correctly on the frontend. Refactoring the tab logic into a centralized tabsConfig file helped ensure that core PIES features and leadership development tools remain accessible to all users, regardless of their assigned role. This structural change reinforced baseline support for every participant while allowing role-specific functionality to scale cleanly.